### ISSUE TITLE
Physical toy caters to the environment of a pediatric patient.

### ISSUE THREAD

**Issue by skalra3** [IITIDIDX597/sp_2019_team2#23]

**Comment by jshah79** [comment]
Story: Physical toy caters to the environment of a pediatric patient. 

As a pediatric patient, 
I want the toy to be accessible on my bedside,
so that I can even lie down and play

Sub Hill 7: If required, there would be printed instructions for both patient and the caregiver/parent to help where needed.

**Comment by Tombodil** [comment]
Story: Physical toy caters to the environment of a pediatric patient.

As a pediatric patient,
I want the toy to be accessible on my bedside,
so that I can even lie down and play

Sub Hill 7: If required, there would be printed instructions for both patient and the caregiver/parent to help where needed. (I feel like this is separate from the story written.)

Hill: (?)

Note: I'm not sure based on whats written, how the story above (overall ok) relates to the sub hill below. This needs to be addressed as the connection to the main hill is also needed information here.

**Comment by akallish** [comment]
@skalra3 @IDziqinwang @jshah79 I agree with @Tombodil that the sub-hill statement and the story do not seem to match.

Here is an iteration of the story:

**As a** pediatric patient,
**I want** Rube-E to be accessible while I am in bed
**So that** I can even lie down and interact with it

**Comment by IDziqinwang** [comment]
Story: Physical toy caters to the environment of a pediatric patient.

**As a** pediatric patient,
**I want** Rube-E to be accessible while I am in bed
**So that** I can even lie down and interact with it directly without asking anyone to help me get it  

Sub Hill 6: The toy would be used by in-patient on their bedside and they can engage 
with it alone without requiring anyone help.

**Comment by skalra3** [comment]
Acceptance criteria:

"Given I'm a pediatric patient,
When I lie down on the bed and want to play the toy
Then I can get in touch with Rube-E because it is within the area where I can reach."
